.travis.yml

```diff
@@ -51,7 +51,7 @@ script:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then export LD_LIBRARY_PATH=$LIBRARY_PATH ; fi
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then cargo build --features vulkan; else cargo build; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then cargo build --features metal; else cargo build; fi
-  - cargo test --all
+  - cargo test --all --features serialize
```

Comment:
Just a thought, you can leave it as is. We could also add the extra test later, should we implement anything more advanced.